lib/transfer_util.cu

```diff
@@ -704,23 +702,24 @@ namespace quda {
 
   template<typename Float>
   void BlockOrthogonalize(ColorSpinorField &V, int Nvec, 
```

Comment:
Excellent.  Probably worthwhile adding them to prolongator and restrictor as well I guess...